flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+  "only-arches": [
+    "x86_64"
```

Comment:
this is disappointing since this is in Python.

Reply:
so maybe this can be removed now?

Reply:
No, I'm not sure how I will get this to build without using some wheels